package.json

```diff
@@ -37,7 +44,8 @@
     "librelay-web": "github:ForstaLabs/librelay-web#a1c22a19b351aae",
     "libsignal-protocol": "github:ForstaLabs/libsignal-protocol-javascript#3a8629b6508",
     "morgan": "1.9.0",
-    "semantic-ui": "2.2.14"
+    "semantic-ui": "2.2.14",
+    "electron": "^1.8.2"
```

Comment:
Let's make this a dev dep so the heroku build doesn't have to pull in a mountain of deps to build.